tendrl/commons/objects/node_context/__init__.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, machine_id=None, node_id=None, fqdn=None,
         self.machine_id = machine_id or self._get_machine_id()
         self.node_id = node_id or self._get_node_id() or self._create_node_id()
         self.fqdn = fqdn or socket.getfqdn()
-        self.tags = tags or NS.config.data['tags']
+        self.tags = tags or NS.config.data.get('tags') or []
```

Comment:
Ok but, I felt the tags are required when executing jobs but the problem is for those which don't themselves execute any jobs performance-monitoring for example.. Without, this change we will then need dummy tag or None equivalent tag.. Please suggest